src/app/Login/page.module.css

```diff
@@ -1,17 +1,21 @@
+.loginPageContainer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 100vh;
+  width: 100vw;
+}
+
 .EmailContainer {
   align-items: center;
-  width: 324px;
-  height: 48px;
-  margin-bottom: 80px;
+  margin-bottom: 2rem;
 }
 .PasswordContainer {
   align-items: center;
-  width: 324px;
-  height: 48px;
-  margin-bottom: 140px;
+  margin-bottom: 4rem;
 }
 .label {
-  margin-bottom: 10px;
+  margin-bottom: 0.1rem;
   display: flex;
 }
 .span {
```

Comment:
instead of having `margin-top: 20px;` and `margin-bottom: 20px`, put it onto one line iwth the shorthand property, `margin: 20px 0;`